includes/class-airtable-api.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
I'd like to see some useful code comments in this file please so we can see what the intention behind it is, including method docblocks just like you have with the other PHP files in this project (which are great!)